kerrors/doc.go

```diff
@@ -14,5 +14,13 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package kerrors contains universally accessible error variables used in Klaytn.
+/*
+Package kerrors contains common error variables used in Klaytn.
+
+Error variables used in multiple packages are declared in kerrors package.
```

Comment:
This sentence does have the same information with the first sentence. How about removing this line?

Reply:
reflected